canonical-websites/partners.ubuntu.com
# Update ./run to latest version

Update run script to latest version from https://github.com/canonical-webteam/generator-canonical-webteam

## QA
Try using various options in the run script but specifically:

- `./run serve --detach` - Should run server in background, for demo use
- Visit http://localhost:8003 and verify
- `./run clean` - Should all be cleaned up
